client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -114,8 +112,7 @@ export const PrescriptionLineEditForm: React.FC<
     []
   );
   const isDirectionsDisabled = !issueUnitQuantity;
-  const displayInDoses =
-    !!OMSPrefs?.displayVaccinesInDoses && !!item?.isVaccine;
+  const displayInDoses = !!OMSPrefs?.manageVaccinesInDoses && !!item?.isVaccine;
```

Comment:
We're allowing dispensing in doses, so this can be renamed too

```suggestion
  const manageVaccinesInDoses = !!OMSPrefs?.manageVaccinesInDoses && !!item?.isVaccine;
```

Reply:
Updated both instances here: 038c1fff43ddff89d43eec0ceff09e804211f015